demo/src/actions/app-actions.js

```diff
@@ -41,13 +56,17 @@ export const loadData = (owner, dataArr) => {
       return;
     }
 
-    let resultData = [];
-    let resultMeta = {};
     const isArray = Array.isArray(dataArr);
```

Comment:
Nit: the name`dataArr` bothers me. If we really wanted to give a hint that this must be an array, then we should type it out and call the parameter `dataArray`. However the first thing we do in the function is to handle non-array values so I think it should just be called `data`.